xbmc/platform/win32/WIN32Util.h

```diff
@@ -75,6 +75,7 @@ class CWIN32Util
   static std::string WUSysMsg(DWORD dwError);
   static bool SetThreadLocalLocale(bool enable = true);
 
+  static std::wstring GetDisplayFriendlyName(const std::wstring& GdiDeviceName);
```

Comment:
Please, can you move this at end of file, after `GetVideoDriverInfo`  (no empty line is necessary)

No particular order is followed here, but it looks a bit confusing attached to the  `// HDR display support comment` without separation.

Reply:
sure